helix-view/src/document.rs

```diff
@@ -576,7 +576,7 @@ impl Document {
     }
 
     /// Detect the indentation used in the file, or otherwise defaults to the language indentation
-    /// configured in `languages.toml`, with a fallback to 4 space indentation if it isn't
+    /// configured in `languages.toml`, with a fallback to the default indentation if it isn't
```

Comment:
This should either say what the default value is ("tabs") or link to `DEFAULT_INDENT`

Reply:
Hmm. Had issues linking the variable, so went with "tabs". It was complaining that it was private, but then still complained when I ran with the respective flag.